src/components/deviceIcons/deviceIcons.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
There is not reason for this component to exist (at least as a separate file). It is only used once. Place this logic in the component where it is used. 

Also, you can simplify this logic a bit by mapping deviceTypes to svg paths.

```
const deviceTypeMappings = {
   Chiller: ChillerSvg,
   Elevator: ElevatorSvg,
   ...
};

// In the function
const svg = deviceTypeMappings[deviceType] || DeviceIconSvg;
```